script.js

```diff
@@ -0,0 +1,641 @@
+      'en upper': 'X',
+      'ru lower': 'ч',
+      'ru upper': 'Ч',
+    },
+    {
+      key: 'KeyC',
+      'en lower': 'c',
+      'en upper': 'C',
+      'ru lower': 'с',
+      'ru upper': 'С',
+    },
+    {
+      key: 'KeyV',
+      'en lower': 'v',
+      'en upper': 'V',
+      'ru lower': 'м',
+      'ru upper': 'М',
+    },
+    {
+      key: 'KeyB',
+      'en lower': 'b',
+      'en upper': 'B',
+      'ru lower': 'и',
+      'ru upper': 'И',
+    },
+    {
+      key: 'KeyN',
+      'en lower': 'n',
+      'en upper': 'N',
+      'ru lower': 'т',
+      'ru upper': 'Т',
+    },
+    {
+      key: 'KeyM',
+      'en lower': 'm',
+      'en upper': 'M',
+      'ru lower': 'ь',
+      'ru upper': 'Ь',
+    },
+    {
+      key: 'Comma',
+      'en lower': ',',
+      'en upper': '<',
+      'ru lower': 'б',
+      'ru upper': 'Б',
+    },
+    {
+      key: 'Period',
+      'en lower': '.',
+      'en upper': '>',
+      'ru lower': 'ю',
+      'ru upper': 'Ю',
+    },
+    {
+      key: 'Slash',
+      'en lower': '/',
+      'en upper': '?',
+      'ru lower': '.',
+      'ru upper': ',',
+    },
+    {
+      key: 'ArrowUp',
+      'en lower': '&#9650;',
+      'en upper': '&#9650;',
+      'ru lower': '&#9650;',
+      'ru upper': '&#9650;',
+    },
+    {
+      key: 'ShiftRight',
+      'en lower': 'Shift',
+      'en upper': 'Shift',
+      'ru lower': 'Shift',
+      'ru upper': 'Shift',
+    },
+  ],
+  [
+    {
+      key: 'ControlLeft',
+      'en lower': 'ctrl',
+      'en upper': 'ctrl',
+      'ru lower': 'ctrl',
+      'ru upper': 'ctrl',
+    },
+
+    {
+      key: 'MetaLeft',
+      'en lower': 'Win',
+      'en upper': 'Win',
+      'ru lower': 'Win',
+      'ru upper': 'Win',
+    },
+    {
+      key: 'AltLeft',
+      'en lower': 'alt',
+      'en upper': 'alt',
+      'ru lower': 'alt',
+      'ru upper': 'alt',
+    },
+    {
+      key: 'Space',
+      'en lower': ' ',
+      'en upper': ' ',
+      'ru lower': ' ',
+      'ru upper': ' ',
+    },
+    {
+      key: 'AltRight',
+      'en lower': 'alt',
+      'en upper': 'alt',
+      'ru lower': 'alt',
+      'ru upper': 'alt',
+    },
+    {
+      key: 'ArrowLeft',
+      'en lower': '&#9668;',
+      'en upper': '&#9668;',
+      'ru lower': '&#9668;',
+      'ru upper': '&#9668;',
+    },
+    {
+      key: 'ArrowDown',
+      'en lower': '&#9660;',
+      'en upper': '&#9660;',
+      'ru lower': '&#9660;',
+      'ru upper': '&#9660;',
+    },
+    {
+      key: 'ArrowRight',
+      'en lower': '&#9658;',
+      'en upper': '&#9658;',
+      'ru lower': '&#9658;',
+      'ru upper': '&#9658;',
+    },
+    {
+      key: 'ControlRight',
+      'en lower': 'ctrl',
+      'en upper': 'ctrl',
+      'ru lower': 'ctrl',
+      'ru upper': 'ctrl',
+    },
+
+
+  ],
+
+];
+const controls = ['Tab', 'CapsLock', 'ShiftLeft', 'ShiftRight', 'ControlLeft', 'MetaLeft', 'AltLeft', 'Space', 'AltRight', 'ControlRight', 'Backspace', 'Enter', 'Delete'];
```

Comment:
I assume you shouldn't have this config in script.js as you have separate config.js file...